plot.c

```diff
@@ -1464,6 +1464,33 @@ void request_to_draw_cells_behind_numeric_input(void)
   redraw_request |= REDRAW_CELLS;
 }
 
+static void frequency_string(char *buf, size_t len, int32_t freq)
+{
+  int l, i;
+  if (freq < 0) {
+    freq = -freq;
+    *buf++ = '-';
+    len -= 1;
+  }
+  if (freq < 1000) {
+    chsnprintf(buf, len, "%d Hz", (int)freq);
+  } else if (freq < 1000000) {
+    chsnprintf(buf, len, "%d.%03d kHz",
+             (int)(freq / 1000),
+             (int)(freq % 1000));
+  } else {
+    chsnprintf(buf, len, "%d.%03d %03d MHz",
+             (int)(freq / 1000000),
+             (int)((freq / 1000) % 1000),
+             (int)(freq % 1000));
+  }
+  l = strlen(buf);
+  for (i=l; i<len-1; i++) {
+    buf[i]=' ';
+  }
+  buf[len-1]=0;
+  return(0);
```

Comment:
Why does this void function return 0?

Reply:
This function does not need to return a value, I will fix it after the merge.

> Why does this void function return 0?

This function does not need to return a value, I will fix it after the merge.